trade-finance-manager-ui/server/types/view-models/record-correction/record-correction-request-information-view-model.ts

```diff
@@ -0,0 +1,14 @@
+import { BaseViewModel } from '../base-view-model';
+
+export type RecordCorrectionRequestInformationViewModel = BaseViewModel & {
+  bank: { name: string };
+  formattedReportPeriod: string;
+  reportId: string;
+  feeRecordId: string;
+  facilityId: string;
+  exporter: string;
+  reasonForRecordCorrection: string;
+  moreInformation: string;
+  contactEmailAddress: string;
+  cancelLink: string;
```

Comment:
Seems to be some repetition in 3/4 view model types sharing common things.

Not for this PR - wondering if we can/should create a new type/model that extends `BaseViewModel` so each specific view model has only unique properties

Not sure if it would convolute things though.